app/posts/manage-teacher-training-applications/2023-03-14-helping-users-decide-if-candidates-meet-the-itt-criteria.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Helping users decide if candidates meet the initial teacher training criteria
+description: Why we decided to simplify the list of reasons for rejecting a candidate, and what the research told us
+date: 2023-03-14
+tags:
+  - rejections
+ogImage:
+  src: /manage-teacher-training-applications/helping-users-assess-candidates-against-itt-criteria/itt-criteria-flow.png
+  alt: "Illustration containing 5 screens: Each screen has a question relating to the initial teacher training criteria. If users answer 'Yes' to every question, they go to a page to give a reason for rejecting the candidate. If users answer 'No' to one or more of the questions, they go to a page where they can give optional feedback to the candidate."
+---
+
+![Illustration containing five screens: The first screen has a heading of 'Degree criteria' followed by a green arrow to screen 2 with a heading of 'English GCSE criteria', followed by a green arrow to screen 3 with a heading of 'Maths GCSE criteria', followed by a green arrow to screen 4. Screen 4 has a heading of 'Reason for rejection'. Users would land on this screen if all their previous answers are 'Yes'. Screen 5 has a heading of 'Optional reasons for rejection'. Users would land on this page if they answered 'No' to one of more of the questions on screen 1, 2 and 3.](itt-criteria-flow.png)
+
+Since the 2022 to 2023 recruitment cycle started in October 2022, we’re seeing more candidates being rejected and fewer providers making offers.
+
+The second highest reason training providers use to reject candidates is ‘qualifications’.
+
+Candidates need to have the following qualifications to do initial teacher training (ITT) in England:
+
+* an undergraduate degree from a UK higher education institution or equivalent qualification
+* a standard equivalent to grade 4 or C GCSE in English and maths
+* a standard equivalent to grade 4 or C GCSE in a science subject (if they’re applying to teach primary)
+
+[The ITT criteria is published for training providers on GOV.UK](https://www.gov.uk/government/publications/initial-teacher-training-criteria/initial-teacher-training-itt-criteria-and-supporting-advice).
+
+## The issue
+
+Previous reporting showed that candidates who are rejected based on their qualifications, are often from outside the UK and sometimes hold equivalent qualifications.
+
+We also saw that eligible candidates are sometimes rejected for reasons outside their control. For example, a candidate applies to a History course, but they have a degree in science.
+
+We believe there is a lot of interpretation and ambiguity in the ITT criteria. This leads to training providers being unsure on how to assess candidates, especially if they have equivalent qualifications.
+
+We wanted to test provider’s understanding of the ITT criteria so that we can better support them when they make decisions about candidates.
```

Comment:
Plural apostrophe here? 🤔 

```suggestion
We wanted to test providers’ understanding of the ITT criteria so that we can better support them when they make decisions about candidates.
```